README.md

```diff
@@ -1,62 +1,66 @@
 # Neo4j Browser
 
-Neo4j Browser is the general purpose user interface for working with Neo4j. Query, visualize, administrate and monitor the database
-with modern and easy-to-use tools.
```

Comment:
Not sure we need to remove all these terms, especially maybe we should keep the query & visualize parts

Reply:
Done.